tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -1,40 +1,69 @@
 ---
-title: Probabilistic Principal Component Analysis
+title: Probabilistic Principal Component Analysis (p-PCA)
 permalink: /:collection/:name/
 weave_options:
   error : false
 ---
 
-Principal component analysis is a fundamental technique to analyse and visualise data.
-You may have come across it in many forms and names.
-Here, we give a probabilistic perspective on PCA with some biologically motivated examples.
-For more details and a mathematical derivation, we recommend Bishop's textbook (Christopher M. Bishop, Pattern Recognition and Machine Learning, 2006).
+# Overview of PCA
 
-The idea of PCA is to find a latent variable $z$ that can be used to describe hidden structure in our dataset.
-We use a simple Gaussian prior for
+Principal component analysis is a fundamental technique to analyse and visualise data. <span style="color:green">It is an unsupervised learning method mainly used for dimensionality reduction. </span>.
+
+For example, we have a feature matrix $A_{m \times n}$, and we would like to extract $d$ principal components which captures most of the information from the matrix but reside in low dimensions (i.e. $d<D$). The aim is to  understand $A_{m \times n}$ through a lower dimensional subspace (e.g. two-dimensional subspace where visualisation is convenient) spanned by the principal components. Geometrically, SVD decomposes the action of $A$ into rotation-scaling-rotation.
```

Comment:
@storopoli many thanks for the thinking. Yes, you are correct:
1. bold for matrix
2. $d<n$ instead of $d<D$ (I initially used D to denote the dimension of A, but later decided to use $n$ instead, and forgot to replace it, sorry).